src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/XamlLspServiceProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml;
+
+[Export(typeof(XamlLspServiceProvider)), Shared]
+internal class XamlLspServiceProvider : AbstractLspServiceProvider
```

Comment:
don't love this.  abstract types shouldn't directly be accessed outside of roslyn.

Reply:
I just verified it's not needed at all.

Reply:
Gone